src/components/site-header/SiteHeader.jsx

```diff
@@ -1,64 +1,26 @@
 import React, { useContext } from 'react';
-import { Link, NavLink } from 'react-router-dom';
 import { HashLink } from 'react-router-hash-link';
 import { AppContext } from '@edx/frontend-platform/react';
 import { getConfig } from '@edx/frontend-platform/config';
 import { Menu as MenuIcon } from '@edx/paragon/icons';
 import { Container, MediaQuery } from '@edx/paragon';
-import edXLogo from '@edx/brand/logo.svg';
 
+import SiteHeaderLogos from './SiteHeaderLogos';
+import SiteHeaderMenu from './SiteHeaderMenu';
 import { Menu, MenuTrigger, MenuContent } from './menu';
 import AvatarDropdown from './AvatarDropdown';
 
 export default function SiteHeader() {
   const config = getConfig();
   const { enterpriseConfig } = useContext(AppContext);
 
-  const renderLogo = () => {
-    const image = (
-      <img
-        className="d-block logo"
-        src={enterpriseConfig.branding.logo || edXLogo}
-        alt={`${enterpriseConfig.name} logo`}
-        data-testid="header-logo-image-id"
-      />
-    );
-    if (enterpriseConfig.disableSearch) {
-      return image;
-    }
-    return (
-      <>
-        <Link to={`/${enterpriseConfig.slug}`} data-testid="header-logo-link-id">
-          {image}
-        </Link>
-      </>
-    );
-  };
-
-  const renderMainMenu = () => {
-    const mainMenuLinkClassName = 'nav-link';
-    if (enterpriseConfig.disableSearch) {
-      return null;
-    }
-    return (
-      <>
-        <NavLink to={`/${enterpriseConfig.slug}`} className={mainMenuLinkClassName} exact>
-          Dashboard
-        </NavLink>
-        <NavLink to={`/${enterpriseConfig.slug}/search`} className={mainMenuLinkClassName} exact>
-          Find a Course
-        </NavLink>
-      </>
-    );
-  };
-
   const renderDesktopHeader = () => (
     <header className="site-header-desktop">
       <Container size="lg">
         <div className="nav-container position-relative d-flex align-items-center">
-          {renderLogo()}
+          <SiteHeaderLogos enterpriseConfig={enterpriseConfig} />
```

Comment:
[nit/curious] Should `SiteHeaderLogos` and `SiteHeaderMenu` pull from the `AppContext` to get the `enterpriseConfig` instead of prop drilling?

Reply:
It would be nicer to look at. It is drilling down just one level and probably won't be used anywhere else outside of `SiteHeader`. I'd be in favor of the change.